shared/src/packages/orders/libs/types/order-status.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type ValueOf } from '~/libs/types/value-of.type.js';
+
+import { type OrderStatus } from '../enums/enums.js';
```

Comment:
why with type ?
```suggestion
import { OrderStatus } from '../enums/enums.js';
```

Reply:
there is arule for eslint that not allows to import without type: All imports in the declaration are only used as types. Use `import type`.eslint[@typescript-eslint/consistent-type-imports](https://typescript-eslint.io/rules/consistent-type-imports)